examples/with-tigris/pages/_document.tsx

```diff
@@ -0,0 +1,26 @@
+import { Html, Head, Main, NextScript } from 'next/document'
```

Comment:
This file should be removed, seems unnecessary for the sake of the example

Reply:
removed